src/app/index.js

```diff
@@ -1,27 +1,33 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
-import { Provider } from 'react-redux';
-import { createStore } from 'redux';
-import { createBottomTabNavigator } from 'react-navigation';
+import { createBottomTabNavigator, createStackNavigator } from 'react-navigation';
 
-import EventScreen from '../Event/components/Event';
 import TimelineScreen from '../Timeline/components/Timeline';
 import SettingsScreen from '../Settings/components/Settings';
 import EventPageContainer from '../EventPage/screens/EventPageContainer';
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#ffffff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-});
+import LoginScreen from '../Login/components/Login';
+import SignupScreen from '../Signup/components/Signup';
 
 const RootNavigator = createBottomTabNavigator({
   Timeline: { screen: TimelineScreen },
   Event: { screen: EventPageContainer },
   Settings: { screen: SettingsScreen },
+},
+{
+  initialRouteName: 'Event',
+});
+
+const LoginNavigator = createStackNavigator({
+  Login: { screen: LoginScreen },
+  Signup: { screen: SignupScreen },
+  Root: { screen: RootNavigator },
+},
+{
+  navigationOptions: {
+    headerLeft: null,
+  },
+},
+{
+  initialRouteName: 'Login',
 });
 
-export default RootNavigator;
+export default LoginNavigator;
```

Comment:
Okay so I think our top-level navigator needs to be called RootNavigator (what you currently have as LoginNavigator) and should live this file. 

What we currently have as RootNavigator I think we should rename something like "HomeNavigator". This means that you'll probably have to change all of the references to `this.props.navigation.navigate('Root')` to refer to `Home` instead.

As for file structure, for now we can keep these two navigators in this file, but I want to think about making a new folder for navigators in the future